.changeset/odd-suns-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lion/overlays': minor
+---
+
+BREAKING: elevation property setter on OverlayController accepts numbers only, previously this was a number as a string. This syncs it with the getter which returns a number.
```

Comment:
Nice, thanks.

I think we can regard it a bugfix, though? Since getter/setter should have been the same type (that's a bug) and providing a string will still work in runtime (not breaking in that regard).

Apart from that, thanks for all the work and I will approve it :)

Reply:
Hmm it's a public API type change, I'd say a breaking bugfix. I guess it doesn't matter much since it's bundled in the same breaking minor release of @lion/overlays anyway, so might as well clarify that this could potentially break a user's code (types-wise).

Reply:
Depends on how you look at it, indeed. Since we need the minor release anyways, it's not a huge deal/difference :)